test/hotspot/gtest/classfile/test_placeholders.cpp

```diff
@@ -39,10 +39,10 @@ TEST_VM(PlaceholderTable, supername) {
   ThreadInVMfromNative tivfn(THREAD);
 
   // Assert messages assume these symbols are unique, and the refcounts start at one.
-  TempNewSymbol A = SymbolTable::new_symbol("abc2_8_2023_class");
-  TempNewSymbol D = SymbolTable::new_symbol("def2_8_2023_class");
+  Symbol* A = SymbolTable::new_symbol("abc2_8_2023_class");
+  Symbol* D = SymbolTable::new_symbol("def2_8_2023_class");
   Symbol* super = SymbolTable::new_symbol("super2_8_2023_supername");
-  TempNewSymbol interf = SymbolTable::new_symbol("interface2_8_2023_supername");
+  Symbol* interf = SymbolTable::new_symbol("interface2_8_2023_supername");
```

Comment:
This doesn't seem like the right way to update this test.  Doesn't this leave the symbols dangling?
And in the face of potential queue draining, it seems to me this could lead the test to intermittent failures.

Reply:
Updated to use the same 'create then drain' approach as the other test.